ipynb/android/workloads/Android_Recents_Fling.ipynb

```diff
@@ -292,7 +292,7 @@
     "    # Allow the activity to start\n",
     "    sleep(5)\n",
     "    # Reset framestats collection\n",
-    "    target.execute('dumpsys gfxinfo com.android.systemui reset')\n",
```

Comment:
This means that before we was also limiting the stats just to the "com.android.systemui" app?
If that's the case, well done! It's better to remove this constraint and to consider system-wide stats ;-)